client/src/components/Input/ModalInput.jsx

```diff
@@ -19,7 +21,9 @@ const Input = ({
           placeholder={placeholder}
           size={size}
           value={value}
+          fontSize={fontSize}
           onChange={onChange}
+          onKeyPress={onKeyPress}
           height={height}
           focused={focused}
```

Comment:
```suggestion
          placeholder={placeholder}
          size={size}
          value={value}
          fontSize={fontSize}
          onChange={onChange}
          onKeyPress={onKeyPress}
          height={height}
          focused={focused}
          {...props}
```
이렇게만 넣어줘도 다른 기본 html input 태그 props들은 설정 따로 안해도 바로 넘겨줄 수 있을 것 같아용 (예를 들면 지금 type, placeholder, value, onKeyPress등은 다 기본 props니까용!)